airflow/migrations/versions/0080_2_0_2_change_default_pool_slots_to_1.py

```diff
@@ -35,8 +35,24 @@
 airflow_version = '2.0.2'
 
 
+# Minimal required schema
+task_instance = sa.table(
+    'task_instance',
+    sa.column('pool_slots', sa.Integer),
+)
+
+
 def upgrade():
     """Change default ``pool_slots`` to ``1`` and make pool_slots not nullable"""
+    connection = op.get_bind()
+    sessionmaker = sa.orm.sessionmaker()
+    session = sessionmaker(bind=connection)
+
+    session.query(task_instance) \
+        .filter(task_instance.c.pool_slots.is_(None)) \
+        .update({task_instance.c.pool_slots: 1}, synchronize_session=False)
+    session.commit()
```

Comment:
It’s much easier to just write SQL here

```suggestion
    op.get_bind("UPDATE task_instance SET pool_slots = 1 WHERE pool_slots IS NULL")
```

Reply:
I wanted to get back to old version and don't write raw SQL, but this is probably simple enough case.
Also I don't see `get_bind` with parameters in the docs, did you mean `execute`?
```suggestion
    op.execute("UPDATE task_instance SET pool_slots = 1 WHERE pool_slots IS NULL")
```

Reply:
Oh yeah it’s `execute`, sorry for the typo.

Reply:
Updated in https://github.com/apache/airflow/pull/24585/commits/f7091f70290196d0f6f7622b056dc831209e891d
(and also rebased to latest main, so CI is happy)